### ISSUE TITLE
README stale link

### ISSUE THREAD

**Issue by yhvh** [cornet/ccze#9]
http://bonehunter.rulez.org/CCZE.html is stale link and should be replaced

**Comment by ansiblejunky** [comment]
Agree! Is there another doc site for this product? It's really a cool software.
